cconv.c

```diff
@@ -456,6 +457,7 @@ void convert_thread_options_to_net(struct thread_options_pack *top,
 	top->stats = cpu_to_le32(o->stats);
 	top->fadvise_hint = cpu_to_le32(o->fadvise_hint);
 	top->fallocate_mode = cpu_to_le32(o->fallocate_mode);
+	top->prepopulate = cpu_to_le32(o->prepopulate);
```

Comment:
We may need to bump `FIO_SERVER_VER` in server.h depending on if a new fio release is done before this goes in. Hard to say at this point! :-P

Reply:
I believe it's not needed if we move to use io_ops instead of the parameter.